src/towncrier/test/test_check.py

```diff
@@ -30,6 +31,29 @@ def create_project(pyproject_path):
     call(["git", "checkout", "-b", "otherbranch"])
 
 
+def commit(message):
+    call(["git", "add", "."])
+    call(["git", "commit", "-m", message])
+
+
+def write(path, contents):
+    dir = os.path.dirname(path)
+    if dir:
+        try:
+            os.makedirs(dir)
+        except OSError:
+            pass
+    with open(path, "w") as f:
+        f.write(contents)
+
+
+def initial_commit():
```

Comment:
A comment would be nice. It's  not clear why existing `create_project` is not good enough for testing.
If you need a different branch name, I think that you can extend create_project and add an optional argument.

Reply:
See https://github.com/twisted/towncrier/pull/337/files/9f705991bea4447da623247db2d4ad3a1f831932#r608653671